src/app/volume/volume-creation/volume-creation-dialog.component.ts

```diff
@@ -18,14 +18,14 @@ export class VolumeCreationDialogComponent {
   @Input() public isLoading: boolean;
   @Input() public diskOfferings: Array<DiskOffering>;
   @Input() public zones: Array<Zone>;
-  @Input() public maxSize: number;
+  @Input() public storageAvailable: string;
```

Comment:
Is there a good reason for it to be a string?

Reply:
it is because in account model primarystorageavailable is a string
Also, this field may have value: 'Unlimited'